src/ztex_sha256crypt.c

```diff
@@ -76,9 +76,9 @@ static struct device_bitstream bitstream = {
 	// (keys * mask_num_cand)/crypt_all_interval per jtr_device.
 	1,			// set by init() base on john.conf setting
 	4096,		// keys/fpga for self-test
-	600 * 1024,	// Would be ~20 MB of USB traffic on 32-byte keys
+	565248,		// Would be ~20 MB of USB traffic on 32-byte keys
 	512,		// Max. number of entries in onboard comparator.
-	150,		// Min. number of keys for effective device utilization
+	23 * 12,	// Min. keys for effective device utilization
 	1, { 135 },	// Programmable clocks
```

Comment:
For consistency, we should set the default frequency to 175 in here as well - not only in default `john.conf`.